README.md

```diff
@@ -14,79 +14,93 @@ These components are setup with the install process available in this repository
+#### Configuring the ports in your VM
+Open inbound ports on your security group. Use the table below as a guide. Make sure you add /32 to the *Elastic IP*.
 
-**Note** We have had problems when uploading big files to Virginia (~25GB). If possible, set up your AWS anywhere else but Virginia.
+| Type | Port | Source | Description |
+| --- | --- | --- | --- |
+| HTTP | 80 | 0.0.0.0/0 | |
+| HTTP | 80 | ::/0 | |
+| HTTPS | 443 | 0.0.0.0/0 | |
+| HTTPS | 443 | ::/0 | |
+| All TCP | 0 - 65535 | _Your VM's Elastic IP_ | |
+| All TCP | 0 - 65535 | _Your Security Group ID_ | | 
+| Custom TCP Rule | 9000 | _Your VM's Elastic IP_ | webservice port |
+| Custom TCP Rule | 9200 | _Your VM's Elastic IP_ | Elasticsearch |
+| SSH | 22 | 0.0.0.0/0 | |
 
-### AWS Tasks
 
-Make sure you do the following:
+#### <a name="makeip"></a>Create and assign an _Elastic IP_ for your VM 
+1. Go to _AWS console_, _Compute_,  _EC2 Dashboard_. There click *running instances*. 
+2. Find your VM and make it active by clicking anywhere in that line.
+3. Under the drop-down *Actions*, go to *Networking*, *Manage IP Addreses*.
+4. Click on *Allocate an Elastic IP*. It will automatically create an IP address and show it on the screen.
+5. Write down that _Elastic IP_. At this point, whatever IP address is shown under IPv4 Public IP for your VM is not the current instance's IP address. Next you need to associate the _Elastic IP_ with your VM.
+6. Back in _EC2 Dashboard_ go to *Elastic IPs*. The IP address just created should be in that list. Check it and under *Actions*, *Associate*, choose resource type "Instance", and choose your EC2 (e.g., searching by its name).
+7.  <a name="sshconnect"></a>In _EC2 Dashboard_ make your VM active by clicking it. Then click _Connect_ on top. The example in that window shows you how to ssh into your VM from a terminal.
 
-* assign an Elastic IP (a static IP address) to your instance
-* open inbound ports on your security group
-    * 80 <- world
-    * 22 <- world
-    * 443 <- world
-    * all TCP <- the elastic IP of the VM (Make sure you add /32 to the Elastic IP)
-    * all TCP <- the security group itself
 
-#### Adding private SSH key to your VM
+#### Adding a private/public key pair to your VM
 
-Add your private ssh key under `~/.ssh/<your_key>.pem`, this is typically the same key that you use to SSH to your host VM, regardless it needs to be a key created on the AWS console so Amazon is aware of it. Then do `chmod 400 ~/.ssh/<your_key>.pem` so your key is not publicly viewable.
+In the VM add your key pair file under `~/.ssh/<your_key_pair>.pem`. This is typically the same key pair that you use to connect to your VM via SSH. This key pair needs to be created on the [AWS console](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html) so Amazon is aware of it. Set the privileges of that key pair file to _read-by-user-only_ by `chmod 400 ~/.ssh/<your_key>.pem` so it is not publicly viewable.
```

Comment:
1. You don't do this in the VM, you do this on your local machine.
2. `This is typically the same key pair that you use to connect to your VM via SSH.`. This seems redundant. The purpose of this section is to describe how to setup your machine to connect to your VM, and, AFAIK, there is no other reason to have the .pem on your local machine.